packages/lib/passport-interface/src/PassportPopup/core.ts

```diff
@@ -0,0 +1,219 @@
+/*
+ * A function that should be called from within a popup window opened by
+ * {@link openZupassPopup}.
```

Comment:
I think the first sentence of this comment should probably be a little higher level and try to download as much relevant information into the reader's brain as fast as possible. e.g. something like:

> call this function on a dedicated `/popup` page in your app to integrate your app with the zupass authentication popup flow. the popup flow is optional, as you could also just redirect to zupass in the same page, but the popup gives zu-auth a more oauth feel, which users are already familiar with.